pkg/internal/venafi/sign.go

```diff
@@ -47,6 +58,30 @@ func (v *Venafi) Sign(csrPEM []byte, duration time.Duration) (cert []byte, err e
 	// Create a vcert Request structure
 	vreq := newVRequest(tmpl)
 
+	// Convert over custom fields from our struct type to venafi's
+	if len(customFields) > 0 {
+		vreq.CustomFields = []certificate.CustomField{}
+		for _, field := range customFields {
+			var fieldType certificate.CustomFieldType
+			switch field.Type {
+			case internalvanafiapi.CustomFieldTypePlain:
+				fieldType = certificate.CustomFieldPlain
+				break
+			case "":
+				fieldType = certificate.CustomFieldPlain
+				break
```

Comment:
nit: `break` isn't required as it is the default behaviour in Go anyway... but 🤷‍♂ it makes no difference!

Reply:
Could also make this `case "", internalvenafiapi.CustomFieldTypePlain:` but again, 🤷‍♂